internal/secrets/encryptor.go

```
@@ -41,6 +48,21 @@ type encryptor struct {
 	// secondaryKey is used during key rotation to provide decryption during key rotations.
 	// It was the primary key that was used for encryption before the key rotation.
 	secondaryKey []byte
+	// primaryKeyHash is prepended to base64-encoded ciphertext with `separator`.
+	primaryKeyHash string
+	// secondaryKeyHash is the previously hash that was prepended to ciphertext with `seperator`
+	secondaryKeyHash string
+}
+
+// sliceKeyHash returns a string which is the first 6 bytes of given key's SHA256 checksum in hexadecimal.
+func sliceKeyHash(k []byte) string {
+	if k == nil {
+		return ""
+	}
+
+	h := sha256.New()
+	h.Write(k)
```

Stdlib is ignoring the error as well: 

https://github.com/golang/go/blob/9706f510a5e2754595d716bd64be8375997311fb/src/crypto/sha256/sha256.go#L257

---

yeah, you can explicitly ignore it with `_ =` to get errcheck off your back.